DevOps.Util.DotNet/Triage/Model.Query.cs

```diff
@@ -79,15 +79,15 @@ private void OnModelCreatingQuery(ModelBuilder modelBuilder)
 
             modelBuilder.Entity<ModelTestResult>()
                 .HasIndex(x => new { x.DefinitionNumber, x.StartTime })
-                .IncludeProperties(x => new { x.BuildResult, x.BuildKind, x.GitHubTargetBranch, x.TestFullName, x.TestRunName, x.IsHelixTestResult });
+                .IncludeProperties(x => new { x.BuildResult, x.BuildKind, x.GitHubTargetBranch, x.TestFullName, x.TestRunName, x.IsHelixTestResult, x.HelixWorkItemName });
```

Comment:
Is there a reason you didn't update the search tests results to also allow querying for this property?

Reply:
Hmm, I didn't really need it, but I can see it being useful (e.g. want to only query for test failures for a specific work item name).  

I think it uses the same indices so it should be easy for me to add it, so I'll take a look.

Reply:
updated
![image](https://user-images.githubusercontent.com/5749229/135352105-c1a43b4c-39d0-4f76-b14c-670deee4e384.png)